johnsonandjohnson/Bodiless-JS
# fix(gatsby-theme-bodiless): Prevent crash when setting image source t…

## Changes
- Do not attempt to create gatsby image nodes for external images

## Test Instructions
1. sttart vital-demo
2. replace hero image with an external image (copy url into source field) & reload page
  - Expected Result: external imge displays properly, but without gatsby image
3. Replace with uploaded image & reload page
  - ER: uploaded image displays as gatsby image.
- Repeat step 2.
- Build and serve site
  - ER: External image displays properly without gatsby image
